api/types.go

```diff
@@ -256,8 +254,6 @@ func (m *Metrics) Summary() {
 	}
 }
 
-var ErrInvalidOpts = fmt.Errorf("invalid options")
```

Comment:
Note: we no longer error if an invalid option is provided (`ollama serve` will log a warning)